elastic/kibana
# [Fleet] Fallback to bundled packages on GET package route

## Summary

Fixes #129998

This fixes both the integration detail page and the integration policy editor for cases where the bundled package version is not available in the registry.

Optimizations made in https://github.com/elastic/kibana/pull/123509 broke this functionality so I have added an explicit test for this case.

### How to test (only valid before 8.2.0 is released)

1. Add [apm-8.2.0.zip](https://github.com/elastic/kibana/files/8473552/apm-8.2.0.zip) to `x-pack/plugins/fleet/target/bundled_packages`
2. Add this to your kibana.yml:
    ```yaml
    xpack.fleet.registryUrl: https://epr.elastic.co
    xpack.fleet.packages:
      - name: apm
        version: latest
    ```
3. Go to `/app/integrations/detail/apm-8.2.0/settings` and ensure APM page is shown without errors
4. Create an APM integration policy and ensure the editor is shown without errors

### Checklist
